jupyter/restore_raft_quorum_with_tablets.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
The PR title looks wrong. It's not a restore procedure but a recovery procedure.